sdk/core/core/test/cmocka/test_json_builder.c

```diff
@@ -47,17 +47,6 @@ void test_json_builder(void** state)
 
     TEST_EXPECT_SUCCESS(az_json_builder_append_object(
         &builder, AZ_SPAN_FROM_STR("int-max"), az_json_token_number(9007199254740991ull)));
-    TEST_EXPECT_SUCCESS(az_json_builder_append_object(
```

Comment:
Even if we are removing `AZ_JSON_TOKEN_SPAN`, this escaping test case and unit test with the expected JSON string **is valid** and we should keep it (re-writing parts of it where needed).